src/components/atoms/charts/Calendar2D/index.jsx

```diff
@@ -0,0 +1,240 @@
+//#region > Imports
+// Contains all the functionality necessary to define React components
+import React from "react";
+//> Additional
+// Used to display popovers on contrib chart items
+import tippy from "tippy.js";
+import "tippy.js/dist/tippy.css";
+// Used to display the time in a readable format
+import moment from "moment";
+//> MDB
+// "Material Design for Bootstrap" is a great UI design framework
+import { MDBRow, MDBCol } from "mdbreact";
+
+//> CSS
+import "./calendar2d.scss";
+//#endregion
+
+const months = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13];
```

Comment:
Add a region and change "months" to "Months".
```suggestion
//#region > Constants
const Months = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13];
//#endregion
```

Reply:
The regions are correct but do not change the variable name with a suggestions. This would crash the whole code.
Btw, it is "MONTHS"
```suggestion
//#region > Constant Variables
const months = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13];
//#endregion
```